2016/proposals/maurya-kumar-raj.md

```diff
@@ -0,0 +1,71 @@
+# Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
+
+## Abstract
+
+The EcoData Retriever is an automated software that is used for finding, downloading, and cleaning up ecological data files, and then organizes the whole data in a local database of your choice. The program cleans and restructures datasets in normalized form and formats them according to the suitable standards of the database management systems.
+
+This project deals with handling the problem of resolving scientific names that are stored in appropriate manner across various datasets. This software would enable its users to conveniently combine different datasets that will be helpful in answering scientific queries. 
+
+## Technical Details
+
+EcoData Retriever is python based tool for downloading, cleaning up, and restructuring ecological data. The information about the repositories are stored in .script files which contain URLs to the files containing the datasets.
+
```

Comment:
For the Description of the retriever you can refer to this :
http://www.ecodataretriever.org/
https://retriever.readthedocs.org/en/latest/introduction.html

I suggest giving a description of the problem: Why do we want to add the taxonomic name resolution system to the EcoData Retriever? What is wrong with the Taxonomic names in the data sets?